bootstrap-vcpkg.sh from master branch fails on s390x alpine linux

**Describe the bug**
`bootstrap-vcpkg.sh` fails when run from master branch on s390x host within alpine container

**Environment**
- OS: Alpine Linux, s390x
- Compiler: revision

**To Reproduce**
Steps to reproduce the behavior:
1. Use Dockerfile below to build alpine devcontainer
2. See error

**Expected behavior**
command is successful and vcpkg binary is available after the command

**Failure logs**

```
 > [build  6/18] RUN cd /tmp     && git clone --branch master --single-branch https://github.com/Microsoft/vcpkg.git     && cd vcpkg     && VCPKG_FORCE_SYSTEM_BINARIES=1 VCPKG_DISABLE_METRICS=1 ./bootstrap-vcpkg.sh:                                                                                                                                                                                                               
0.398 Cloning into 'vcpkg'...                                                                                                                                                                                      
Updating files: 100% (11127/11127), done.                                                                                                                                                                          
153.8 Unable to determine a binary release of vcpkg; attempting to build from source.                                                                                                                              
153.8 Downloading vcpkg tool sources
155.0 Building vcpkg-tool...
155.2 CMake Error: Could not create named generator 'Ninja'
155.2 
155.2 Generators
155.2   Green Hills MULTI            = Generates Green Hills MULTI files
155.2                                  (experimental, work-in-progress).
155.2 * Unix Makefiles               = Generates standard UNIX makefiles.
155.2   Ninja                        = Generates build.ninja files.
155.2   Ninja Multi-Config           = Generates build-<Config>.ninja files.
155.2   Watcom WMake                 = Generates Watcom WMake makefiles.
155.2   CodeBlocks - Ninja           = Generates CodeBlocks project files.
155.2   CodeBlocks - Unix Makefiles  = Generates CodeBlocks project files.
155.2   CodeLite - Ninja             = Generates CodeLite project files.
155.2   CodeLite - Unix Makefiles    = Generates CodeLite project files.
155.2   Eclipse CDT4 - Ninja         = Generates Eclipse CDT 4.0 project files.
155.2   Eclipse CDT4 - Unix Makefiles= Generates Eclipse CDT 4.0 project files.
155.2   Kate - Ninja                 = Generates Kate project files.
155.2   Kate - Unix Makefiles        = Generates Kate project files.
155.2   Sublime Text 2 - Ninja       = Generates Sublime Text 2 project files.
155.2   Sublime Text 2 - Unix Makefiles
155.2                                = Generates Sublime Text 2 project files.
155.2 
------
Dockerfile:20
--------------------
  19 |     ARG VCPKG_BRANCH="master"
  20 | >>> RUN cd /tmp \
  21 | >>>     && git clone --branch $VCPKG_BRANCH --single-branch https://github.com/Microsoft/vcpkg.git \
  22 | >>>     && cd vcpkg \
  23 | >>>     && VCPKG_FORCE_SYSTEM_BINARIES=1 VCPKG_DISABLE_METRICS=1 ./bootstrap-vcpkg.sh
  24 |     
--------------------
ERROR: failed to solve: process "/bin/sh -c cd /tmp     && git clone --branch $VCPKG_BRANCH --single-branch https://github.com/Microsoft/vcpkg.git     && cd vcpkg     && VCPKG_FORCE_SYSTEM_BINARIES=1 VCPKG_DISABLE_METRICS=1 ./bootstrap-vcpkg.sh" did not complete successfully: exit code: 1
```


**Additional context**
Alpine Linux is from a docker container, from following Dockerfile:

```
FROM alpine:3.18.2 as build
LABEL description="Alpine devcontainer"

# ...

RUN apk update && apk add --no-cache \
   autoconf build-base binutils cmake curl file linux-headers make musl-dev musl-dbg ninja-build \
   ninja-is-really-ninja tar unzip zip gzip wget git pkgconfig doxygen clang \
   clang-dev util-linux-dev gcovr gdb cmake openssh-client rpm

# link ninja where expected by vcpkg
RUN ln -s /usr/bin/ninja /usr/local/bin/ninja

ARG VCPKG_BRANCH="master"
RUN cd /tmp \
    && git clone --branch $VCPKG_BRANCH --single-branch https://github.com/Microsoft/vcpkg.git \
    && cd vcpkg \
    && VCPKG_FORCE_SYSTEM_BINARIES=1 VCPKG_DISABLE_METRICS=1 ./bootstrap-vcpkg.sh

...
```

UPDATE: **This was working on 20 Feb 2024**

--- a/scripts/bootstrap.sh
+++ b/scripts/bootstrap.sh
@@ -217,7 +217,7 @@ else
         cmakeConfigOptions=" $cmakeConfigOptions '-DCMAKE_JOB_POOL_COMPILE:STRING=compile' '-DCMAKE_JOB_POOL_LINK:STRING=link' '-DCMAKE_JOB_POOLS:STRING=compile=$VCPKG_MAX_CONCURRENCY;link=$VCPKG_MAX_CONCURRENCY' "
     fi
 
-    (cd "$buildDir" && cmake "$srcDir" $cmakeConfigOptions) || exit 1
+    (cd "$buildDir" && eval cmake "$srcDir" $cmakeConfigOptions) || exit 1
     (cd "$buildDir" && cmake --build .) || exit 1
 
     rm -rf "$vcpkgRootDir/vcpkg"

